my_init.d/create-system-user.sh

```diff
@@ -1,17 +1,47 @@
 #!/bin/bash
 set -em
 
-# Add $SYSTEM_USER user (no password)
-mkdir /home/$SYSTEM_USER                                                         && \
-      useradd --home /home/$SYSTEM_USER --uid $SYSTEM_USER_UID --shell /bin/bash $SYSTEM_USER && \
-      chown -R $SYSTEM_USER:$SYSTEM_USER /home/$SYSTEM_USER
+PERFORM_CHOWN=false
 
-# Add .bashrc file to the $SYSTEM_USER user
-cp -v /etc/skel/.bashrc /etc/skel/.bash_logout /etc/skel/.profile /home/$SYSTEM_USER/
+# Create user's home folder, it should not exist at the beginning.
```

Comment:
would that not logically belong inside the if block below?

Reply:
If one mounts a folder using '-v' option (like JupterHub does), the folder will be created along with mounting